client/src/controllers/InitController.ts

```diff
@@ -14,20 +14,41 @@ import { debounce } from '../utils/debounce';
  * <div class="keep-me hide-me" data-controller="w-init" data-w-init-event-value="custom:event other-custom:event">
  *   When the DOM is ready, two additional custom events will be dispatched; `custom:event` and `other-custom:event`.
  * </div>
+ *
+ * @example - Detail dispatching
+ * <div class="keep-me" data-controller="w-init" data-w-init-detail-value='{"status": "success", "message": "Detail is ready"}'>
+ *  When the DOM is ready, the detail with value of a JSON object: {"status": "success", "message": "Detail is ready"} will be dispatched.
+ * </div>
```

Comment:
Minor tweak here, let's keep this example simpler, no need for the classname nor the repeated JSON text, maybe something a bit different like this;

```suggestion
 * @example - Custom event dispatching with detail
 * <article data-controller="w-init" data-w-init-detail-value='{"status": "success", "message": "Article has entered the room"}'>
 *   When the DOM is ready, the detail with value of a JSON object above will be dispatched.
 * </article>
```